src/java.base/share/classes/sun/launcher/LauncherHelper.java

```diff
@@ -589,6 +590,17 @@ static void printXUsageMessage(boolean printToStderr) {
         }
     }
 
+    /**
+     * Prints the short usage text to the desired output stream.
+     */
+    static void printConciseUsageMessage(boolean printToStderr) {
+        initOutput(printToStderr);
+        ostream.println(SharedSecrets.getJavaLangAccess().shortVersionString());
```

Comment:
What is the reason for printing the short version string at the start of the short usage message?

Reply:
In passing, it may be better to pick "short" or "concise", right now it's a mix in the method and resource keys.

Reply:
The idea behind printing the short version is to identify what tool is speaking. It can be removed, if desired.

Reply:
`shortVersionString` changed to `conciseVersionString` in:
https://github.com/openjdk/jdk/pull/21411/commits/e9e9ad4bdd0a11244cb1ef84fdeea036e94718c7

Reply:
I hadn't paid attention to the version string in the output for `java` (without args). I agree with Alan that it might be better to remove that from the output so that it matches with `java -help` which doesn't print it.